packages/twenty-website/src/content/user-guide/what-is-twenty.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: What is Twenty
+position: 0
+icon: IconUsers
+info: A brief on Twenty's commitment to reshaping CRM with Open Source
+image: /images/user-guide/home/what-is-twenty.png
+---
+
+## What is Twenty
+
+The purpose of this user guide is to help you learn how you can use Twenty to build the CRM you want. 
+
+## Quick Search 
+
+You'll see a search bar at the top of your sidebar. You can also bring up the command bar with the `cmd`/`ctrl` + `k` shortcut to navigate through your workspace, and find people, companies, notes, and more. 
+
+The command bar also supports other shortcuts for navigation.  
+
+## Create Pre-filtered Views 
```

Comment:
FYI this was just dummy text in Figma, we'll create a followup issue to re-write all texts once design is done

Reply:
I added it just to check the working